cms-contentful/it/ContentfulCmsServiceIT.java

```diff
@@ -0,0 +1,68 @@
+import io.commercetools.sunrise.cms.CmsIdentifier;
+import io.commercetools.sunrise.cms.contentful.ContentfulCmsService;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.Optional;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.TimeUnit;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ContentfulCmsServiceIT {
+    private static final Locale DE = Locale.forLanguageTag("de");
+    private static final Locale EN_US = Locale.forLanguageTag("en-US");
+    private static final List<Locale> SUPPORTED_LOCALES = asList(DE, EN_US);
+
+    // credentials for contentful demo account
+    private static final String SPACE_ID = "cfexampleapi";
+    private static final String TOKEN = "b4c0n73n7fu1";
+    private ContentfulCmsService contentfulCmsService;
+
+    @Before
+    public void setUp() throws Exception {
+        contentfulCmsService = ContentfulCmsService.of(SPACE_ID, TOKEN);
+    }
+
+    @Test
+    public void whenAskForExistingStringContentThenGet() throws Exception {
+        CmsIdentifier identifier = CmsIdentifier.ofEntryTypeAndKeyAndField("human", "Finn", "description");
+        CompletionStage<Optional<String>> optionalCompletionStage = contentfulCmsService.get(SUPPORTED_LOCALES, identifier);
+        Optional<String> content = optionalCompletionStage.toCompletableFuture().get(5, TimeUnit.SECONDS);
+
+        assertThat(content.isPresent()).isTrue();
+        assertThat(content.get()).isEqualTo("Fearless adventurer! Defender of pancakes.");
+    }
+
+    @Test
+    public void whenAskForNotExistingStringContentThenNotPresent() throws Exception {
+        CmsIdentifier identifier = CmsIdentifier.ofEntryTypeAndKeyAndField("human", "Jake", "likes");
+        CompletionStage<Optional<String>> optionalCompletionStage = contentfulCmsService.get(SUPPORTED_LOCALES, identifier);
+        Optional<String> content = optionalCompletionStage.toCompletableFuture().get(5, TimeUnit.SECONDS);
+
+        assertThat(content.isPresent()).isFalse();
```

Comment:
as in https://github.com/commercetools/commercetools-sunrise-java-cms/pull/2/files#r64696705 it could be improved by

`assertThat(content).isEmpty();`

http://joel-costigliola.github.io/assertj/core-8/api/org/assertj/core/api/AbstractOptionalAssert.html